go.mod

```diff
@@ -4,19 +4,16 @@ go 1.12
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/blang/semver v3.5.1+incompatible // indirect
 	github.com/buger/goterm v0.0.0-20181115115552-c206103e1f37
 	github.com/checkpoint-restore/go-criu v0.0.0-20190109184317-bdb7599cd87b
-	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/containernetworking/cni v0.7.2-0.20190904153231-83439463f784
 	github.com/containernetworking/plugins v0.8.5
-	github.com/containers/buildah v1.13.1
+	github.com/containers/buildah v1.14.1-0.20200219125159-7cd6f7d04842
```

Comment:
Did you spin up a 1.14.1 somewhere?  The  highest release I see is 1.14.0.

Reply:
Nope this comes from the master branch.  I have no idea where it comes from.

Reply:
I think this is being generated by go.mod somehow.  The go.mod that I added was from the master branch.  We will need to cut a release before podman gets its next release.  But this should be fine for now.

$ ./bin/podman info  | grep -i buildah
BuildahVersion: 1.15.0-dev